stdlib/test/builtin/test_slice.mojo

```diff
@@ -10,18 +10,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ===----------------------------------------------------------------------=== #
-# RUN: %mojo %s | FileCheck %s
+# RUN: %mojo -debug-level full %s
```

Comment:
Similar comment as in the other PRs.  Do you mind removing the `-debug-level full` please?